ClassLibrary/ClassLibrary.Test/TestEntities.cs

```diff
@@ -141,5 +140,33 @@ public void AddEqualBooksInCard()
 
             Should.Throw<ArgumentException>(() => DR.BorrowBook(reader, book));
         }
+
+        [Test]
+        public void DeleteBookInCard()
```

Comment:
why this test is not in DepartmentReaders test class?

Reply:
I prefer being closer to the essence(entity) and not duplicating code. Although sometimes I can contact through DepartmentReaders, which is not very pleasant =(